docs/content-guide/redirects.md

```diff
@@ -17,6 +17,29 @@ Use redirects for fun and profit!
 
 ---
 
+## Background
+
+If you move an existing article in the documentation, you **must** add redirects. Otherwise, links break between versions and users cannot find the information they are looking for, even if it exists in the version they are viewing.
```

Comment:
```suggestion
If you move an existing article in the documentation, you **must** add redirects. Otherwise, links break between versions and users cannot find the information they are looking for, even if it exists in the version they are viewing. Additionally, we also recommend replacing references to the previous URL with your new URL in your documentation text.
```